### DISCUSSION TITLE
Custom Edges with Custom Nodes

### DISCUSSION THREAD

**Discussion by muneeb127** [xyflow/xyflow#2865] [Q&A]
I am using multiple custom nodes for my application and I want to use differet custom edge based on the type of node.
Is there a way to use different type of connection lines based on the type on nodes being connected? I want to implement the logic in onConnect() function but it takes the source id and destination id as the parameter and not the whole node object.

**Comment by moklick** [comment]
Unfortunately not. You would need to search for the node id and grab the type from there.
